launch/launch/utilities/signal_management.py

```diff
@@ -143,6 +142,9 @@ def __on_sigterm(signum, frame):
             # Windows does not support SIGQUIT
             signal.signal(signal.SIGQUIT, __on_sigquit)
     except ValueError:
-        _logger.error("failed to set signal handlers in 'launch.utilities.signal_management.py'")
-        _logger.error('this function must be called in the main thread')
+        logger = logging.getLogger(__name__)
+        logger.error(
+            "failed to set signal handlers in 'launch.utilities.signal_management.py'",
```

Comment:
Why is this multi line is it longer now? Was it too long before? Why does it have a trailing `,`?

Reply:
It got changed at some point before I made these changes. I'll restore it.